README.md

```diff
@@ -286,52 +292,169 @@ type Action = {
   message: {
     body: {
       active: boolean;
-      type: 'like' | 'follow'
+      type: "like" | "follow";
       targetUri: FarcasterURI;
-      schema: 'farcaster.xyz/schemas/v1/action';
-    }; 
+      schema: "farcaster.xyz/schemas/v1/action";
+    };
     account: number;
     timestamp: number;
   };
-}
+};
 ```
 
 #### Message Validation
+
 1. `schema` must be known.
 2. `active`, `type` must be present and match types.
 3. `targetUri` must be a valid FarcasterURI and must not reference this message.
 
 #### Set Construction
 
-Actions are managed with an [LWW-Element-Set CRDT](https://en.wikipedia.org/wiki/Conflict-free_replicated_data_type#LWW-Element-Set_(Last-Write-Wins-Element-Set)) which guarantees strong eventual consistency. Conceptually, there is a single **set** that stores all messages and conflicts are resolved by timestamp and lexicographical hash order. An addition is performed by constructing an `Action` message `a` where `active` is true, while a remove is performed by setting `active` to false. In both cases, the logic for merging the message into the set is: 
+Actions are managed with an [LWW-Element-Set CRDT](<https://en.wikipedia.org/wiki/Conflict-free_replicated_data_type#LWW-Element-Set_(Last-Write-Wins-Element-Set)>) which guarantees strong eventual consistency. Conceptually, there is a single **set** that stores all messages and conflicts are resolved by timestamp and lexicographical hash order. An addition is performed by constructing an `Action` message `a` where `active` is true, while a remove is performed by setting `active` to false. In both cases, the logic for merging the message into the set is:
 
 1. If there is an action `x` in the set with the same values for `type`, `targetUri` and `account` as the incoming action `y`:
    - If `x > y`, discard `y`
    - If `x < y`, delete `x` and add `y` into the set
-2. Otherwise, add `y` into the set. 
-
+2. Otherwise, add `y` into the set.
 
 ## 4.3 Verifications
-*This section is still under development and will cover a CRDT for verified data types, including proof of ownership of an address or a specific NFTs*
+
+Verifications are bi-directional proofs of ownership between Farcaster accounts and external identities. Initially, Ethereum addresses are the only supported external identities, though other blockchains can be added later.
```

Comment:
I don't love the term `external identities` in the first line because we are very likely to have verifications that simply authorize an asset (e.g. Proof of NFT) or something you've achieved offline (Proof of Diploma). A diploma or punk isn't an "identity" per se, but an element of an identity

Reply:
Agreed. I changed it to `external entities`. Can you think of something better?